pkg/liqoctl/status/pods.go

```diff
@@ -33,7 +33,8 @@ const (
 )
 
 var (
-	liqoDeployments = []string{
+	// LiqoDeployments contains the list of Liqo deployments.
```

Comment:
Wouldn't it be appropriate to move this file to status/local, and revert the changes?